test/fixtures/incorrect.js

```diff
@@ -237,6 +237,18 @@ class NoUnderscoreDangle {
 
 noop(new NoUnderscoreDangle());
 
+// `babel/no-unused-expressions`
```

Comment:
Unused ternary expressions are also not allowed, so can you please add an example of that? (Please check if it only throws the `babel/no-unused-expressions` error.)